backend/app/core/aws/refresh_infrastructure_lambda.py

```diff
@@ -0,0 +1,31 @@
+from logging import getLogger
+from typing import Any
+
+from botocore.exceptions import ClientError
+
+from app.core.aws.boto3_clients import get_client
+from app.settings import settings
+
+
+class RefreshInfrastructureLambda:
+    def __init__(self):
+        self.enabled = bool(settings.INFRASTRUCTURE_LAMBDA_ARN)
+        self.lambda_arn = settings.INFRASTRUCTURE_LAMBDA_ARN
+        self.logger = getLogger()
+        if self.enabled:
+            self.client = get_client("lambda")
```

Comment:
```suggestion
    def __init__(self):
        self.lambda_arn = settings.INFRASTRUCTURE_LAMBDA_ARN
        self.logger = getLogger()
        if self.lambda_arn:
            self.client = get_client("lambda")
```
No need to have extra attribute